js/repl/Worker.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import compile from "./compile";
+import registerPromiseWorker from "promise-worker/register";
+
+declare var Babel: any;
+declare var babelPresetEnvStandalone: any;
+declare function importScripts(url: string): void;
+
+// This script should be executed within a web-worker.
+// Values returned below will be automatically wrapped in Promises.
+registerPromiseWorker(message => {
+  const { method } = message;
+
+  switch (method) {
+    case "compile":
+      return compile(message.code, message.config);
+
+    case "getBabelVersion":
```

Comment:
Ooh, I should totally steal this idea rather than building a [custom file](https://github.com/prettier/prettier/blob/master/website/static/lib/prettier-version.js)